### DISCUSSION TITLE
How do I set a schedule in package rules to run the packagerules every friday morning?

### DISCUSSION THREAD

**Discussion by btxbtx** [renovatebot/renovate#18123] [Request Help]
First, [the documentation](https://docs.renovatebot.com/configuration-options/#schedule) gives an example:

```yaml
{
  "packageRules": [
    {
      "matchPackageNames": ["aws-sdk"],
      "extends": ["schedule:monthly"]
    }
  ]
}
```

I want to run some of my packageRules every friday morning.  So I check the available [schedule presents documentation](https://docs.renovatebot.com/presets-schedule/) and it is not clear how to do that.

How do I set a schedule in package rules to run the packagerules every friday morning?  Would it be `schedule:Friday` or `schedule:friday` or `schedule:Fridays` or `scheduleFriday`?   Does `schedule:weekly` default to early Mondays?

It is hard for me to tell based on the documentation, and cumbersome to iterate over the possibilities.

**Comment by rarkins** [comment]
You don't need to use a `schedule:*` preset (via `extends`). You can just use the `schedule` config option directly.

**Comment by RahulGautamSingh** [comment]
Yes `schedule:weekly` defaults to earlyMondays.

You can use the follwoing schedule options for your case

This would mean Renovate will run for 7 hours on Friday morning.
```json
{
  "schedule": ["after 2am and before 9am every friday"]
}
```

Weekly schedule on early Friday mornings.
```json
{
  "schedule": [
    "before 3am on Friday"
  ]
}
```
